_dashboards/management/index-patterns.md

```diff
@@ -0,0 +1,94 @@
+---
+layout: default
+title: Index patterns
+parent: Dashboards Management
+nav_order: 10
+has_children: true
+---
+
+# Index patterns
+Updated 2.10
+{: .label .label-purple }
+
+Index patterns are essential for querying and analyzing data in OpenSearch. They are used to define the fields that are available in the data and the mapping between the fields and their data types. Index patterns are also used to create dashboards and visualizations in OpenSearch Dashboards. 
+
+An index pattern can point to one or more indexes, data streams, or index aliases. For example, an index pattern can point you to your log data from yesterday or all indexes that contain your data. To visualize your data, you need to tell OpenSearch from which indexes to retrieve the data. Because it's a common approach to have data stored in multiple indexes, creating an index pattern enables your visualizations retrieve data from any indexes that match the index pattern. 
+
+The following is a basic example of an index pattern:
+
+```json
+{
+    "index-patterns": ["my-index-*"],
+    "mappings": {
+        "my-field": {
+            "type": "string"
+        }
+    }
+}
+```
+
+This index pattern defines a single field, `my-field`, which is a string field. The index pattern also specifies that the index pattern matches any index that starts with the prefix `my-index`. 
+
+To use this index, you could query for all documents in the index pattern that have the value `my-value` in the `my-field` field, as shown in the following example:
```

Comment:
In dashboards, you can only query with DQL, correct? If so, I would not include JSON here but would add a reference to the DQL page and say that index patterns are used in Discover, visualizations, etc.